ensawards.org/src/components/atoms/badges/MatrixBadge.tsx

```diff
@@ -0,0 +1,115 @@
+import { GenericTooltip } from "@/components/atoms/GenericTooltip.tsx";
+import { ResolveAndDisplayIdentity } from "@/components/atoms/identity";
+import { Skeleton } from "@/components/ui/skeleton";
+import { TooltipProvider } from "@/components/ui/tooltip.tsx";
+import { type BenchmarkResult, type BenchmarkVerifier } from "@/types/apps.ts";
+import { chainIdToNamespaceId } from "@/utils/chains.ts";
+import { getENSNodeUrl } from "@/utils/env";
+import { ENSNodeProvider, createConfig } from "@ensnode/ensnode-react";
+import { ENSNodeClient, type UnresolvedIdentity } from "@ensnode/ensnode-sdk";
+import { Check, CheckCheck, X as XIcon } from "lucide-react";
```

Comment:
```suggestion
import { Check as PassIcon, CheckCheck, X as FailIcon } from "lucide-react";
```

Goal: Make code more self-documenting for our context.

And what is the `CheckCheck` icon? Please rename it accordingly.